docs/index.md

```diff
@@ -0,0 +1,91 @@
+---
+sd_hide_title: true
+hide-toc: true
+---
+
+# AutoGluon-Cloud
+
+::::::{div} landing-title
+:style: "padding: 0.1rem 0.5rem 0.6rem 0; background-image: linear-gradient(315deg, #438ff9 0%, #3977B9 74%); clip-path: polygon(0px 0px, 100% 0%, 100% 100%, 0% calc(100% - 1.5rem)); -webkit-clip-path: polygon(0px 0px, 100% 0%, 100% 100%, 0% calc(100% - 1.5rem));"
+
+::::{grid}
+:reverse:
+:gutter: 2 3 3 3
+:margin: 4 4 1 2
+
+:::{grid-item}
+:columns: 12 4 4 4
+
+```{image} ./_static/autogluon-s.png
+:width: 200px
+:class: sd-m-auto sd-animate-grow50-rot20
+```
+:::
+
+:::{grid-item}
+:columns: 12 8 8 8
+:child-align: justify
+:class: sd-text-white sd-fs-3
+
+AutoGluon-Cloud: Train and Deploy AutoGluon on the Cloud
+
+:::
+::::
+
+::::::
+
+AutoGluon-Cloud aims to provide user tools to train, fine-tune and deploy [AutoGluon](<https://auto.gluon.ai/stable/index.html>) backed models on the cloud. With just a few lines of code, users can train a model and perform inference on the cloud without worrying about MLOps details such as resource management.
+
+Currently, AutoGluon-Cloud supports [AWS SageMaker](<https://aws.amazon.com/sagemaker/>) as the cloud backend.
+
+## {octicon}`rocket` Quick Examples
+
+```python
+from autogluon.cloud import TabularCloudPredictor
+import pandas as pd
+train_data = pd.read_csv("https://autogluon.s3.amazonaws.com/datasets/Inc/train.csv")
+test_data = pd.read_csv("https://autogluon.s3.amazonaws.com/datasets/Inc/test.csv")
+predictor_init_args = {"label": "class"}  # init args you would pass to AG TabularPredictor
+predictor_fit_args = {"train_data": train_data, "time_limit": 120}  # fit args you would pass to AG TabularPredictor
+cloud_predictor = TabularCloudPredictor(cloud_output_path='YOUR_S3_BUCKET_PATH')
+cloud_predictor.fit(predictor_init_args=predictor_init_args, predictor_fit_args=predictor_fit_args)
+cloud_predictor.deploy()
+result = cloud_predictor.predict_real_time(test_data)
+cloud_predictor.cleanup_deployment()
+# Batch inference
+result = cloud_predictor.predict(test_data)
```

Comment:
Having no new-lines here makes this a bit hard to read through. Could we run it through `black`?